HW6 is done

Borisovab opened the pull request:
realm for friends, groups and photos

Borisovab committed:
HW6 is done

realm for friends, groups and photos

AntonChehow0 commented on VKclient/PartOfPresentProjectVersion/RealmData/DataGroupsArrayParam.swift:
```diff
@@ -0,0 +1,36 @@
+
+//MARK: - Data for groups
+
+class RealmResponseGroups: Object {
+    @Persisted
+    var count: Int
+
+    @Persisted
+    var items: List<RealmGroupsArrayParam>
+
+}
```
В этом нет смысла
Храните только данные, а не обвязку, которую вк себе сделал

AntonChehow0 commented on VKclient/WebDataRequest.swift:
```diff
@@ -79,43 +145,194 @@ class WebDataRequest {
+
+//MARK: - method for getAll photo parameters (запаска!!!!!!)
+//
+//func requestPhoto(completion: @escaping (Result<JSONInfo<ResponsePhotos>, Error>) -> Void) {
+//
+//    var components = URLComponents(string: "https://api.vk.com/method/photos.getAll")
+//    components?.queryItems = [
+//        URLQueryItem(name: "access_token", value: Session.sharedInstance.token),
+//        URLQueryItem(name: "owner_id", value: "-121214684"),
+//        URLQueryItem(name: "no_service_albums", value: "1"),
+//        URLQueryItem(name: "v", value: "5.131")
+//    ]
+//    guard let url = components?.url else { return }
+//
+//    URLSession.shared.dataTask(with: url) { data, response, error in
```
Коментарии не нужны

Borisovab committed:
Changed VC News.

Changed VC News. Placed friends in the first section and group news in the second section. Displaying the gallery in all groups only from one group.
